Libraries/Network/__tests__/XMLHttpRequestBase-test.js

```diff
@@ -54,6 +54,25 @@ describe('XMLHttpRequestBase', function(){
 		expect(xhr.readyState).toBe(xhr.OPENED);
     });
 
+    it('should expose responseText correctly', function() {
```

Comment:
Grr, yes. This file is, for some reason, indented with tabs and not spaces. Maybe after this PR I'll fix that because it's super annoying.